.env.example

```diff
@@ -2,3 +2,4 @@ OPENAI_API_KEY=your_openai_api_key_here
 N_IMAGES=5
 MIN_IMAGE_WIDTH=300
 MIN_IMAGE_HEIGHT=250
+ALLOWED_IMAGE_SIZES=300x250,728x90,160x600,300x600
```

Comment:
Remove this

Reply:
Thank you for the feedback! I'll remove the `ALLOWED_IMAGE_SIZES` entry from the `.env.example` file since it's now hard coded. I'll make the necessary changes and push an update shortly.